src/tools/bin/plot_chargeConservation_overTime.py

```diff
@@ -41,36 +38,11 @@
 normalized to the maximum [per-species] charge in the first
 simulation time step.
 
-Developer: Richard Pausch
+Developer: Richard Pausch, Rene Widera
```

Comment:
I am not sure why we have it like that. But there is another author list in the copyright that was different from this one even before this PR

Reply:
I assume it is because this is a doc string for python. If you use your python IDE you see the doc string